crates/bevy_ui/src/node_bundles.rs

```diff
@@ -93,7 +93,7 @@ pub struct ImageBundle {
     /// The transform of the node
     ///
     /// This component is automatically managed by the UI layout system.
-    /// To alter the position of the `ImageBundle`, use the properties of the [`Style`] component.
+    /// To alter the position of the `ImageBundle`, use the properties of the [`Node`] component.
```

Comment:
There need to be stronger warnings even to users against modifying the `Transform` of UI nodes but there doesn't seem to be a natural place anymore with required components for context specific documentation like we could have in bundles. Maybe on the require?
```
#[require(
    Node,
    // A Node's Transform is automatically managed by the UI layout system, manually changing its values should be avoided
    Transform,
     // .. rest of required components
)]
```
It doesn't seem great though and wouldn't show up in doc comments, unless something could be done with the require macro?

Reply:
It's outside the scope of this PR though, can be discussed later.